apps/financial-aid/backend/src/app/modules/application/application.service.ts

```diff
@@ -207,6 +212,15 @@ export class ApplicationService {
     const files = await this.fileService.getAllApplicationFiles(id)
     updatedApplication?.setDataValue('files', files)
 
+    await this.sendEmail(
```

Comment:
Aren't we here sending an email for every update to the application? 

For example when we are adding files and changing employee that is working on the application? In that case no email should be sent.